libs/auth-api-lib/migrations/20240327103605-add-subject-id-to-delegation-index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn(
+        'delegation_index',
+        'subject_id',
+        {
+          type: Sequelize.STRING,
+          allowNull: true,
```

Comment:
Should be required field.

We need to either truncate the current index table, or create the column with allow null, to some one time migration of populating the subject ID, then set the column to allowNull false.